lint/witnesslint/linter.py

```diff
@@ -750,6 +745,12 @@ def handle_graphml_elem(self, graphml_elem):
                 "for 'http://www.w3.org/2001/XMLSchema-instance'",
                 graphml_elem.sourceline,
             )
+        for attr in graphml_elem.attrib.items():
+            if attr[0] != "{http://www.w3.org/2001/XMLSchema-instance}schemaLocation":
+                logging.warning(
+                    "Unexpected attribute on graphml element: {}".format(attr[0]),
```

Comment:
Could you change this to omit the namespace? For consistency with the other warnings it should be like this:
```suggestion
                    "Unexpected attribute on graphml element: {}".format(attr[0].rpartition("}")[2]),
```